tests/dummy/app/pods/size/controller.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * TODO
+ */
+
+import Ember from 'ember'
+const {A, Controller, isEmpty} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {sort} from 'ember-frost-sort'
+
+export default Controller.extend({
+
+  // == Dependencies ==========================================================
+
+  // == Properties ============================================================
+
+  expandedItems: A([]),
```

Comment:
Can be just `A()` (same for all other instances in this file)

Reply:
Good catch, copied from other dummy routes. In order to make this feature PR clean, I'll raise another PR later to clean up the rest of the dummy.